src/conf_mode/protocols_isis.py

```diff
@@ -149,7 +149,7 @@ def verify(isis):
     # If Redistribute set, but level don't set
     if 'redistribute' in isis:
         proc_level = isis.get('level','').replace('-','_')
-        for afi in ['ipv4']:
+        for afi in ['ipv4'] or afi in ['ipv6']:
```

Comment:
Should be: `for afi in ['ipv4', 'ipv6']:`

Reply:
Ahh, sure. ok. Will make the change. Thank you :)